content/standards/360giving/index.html.md

```diff
@@ -1,23 +1,29 @@
 ---
-title: 360Giving Schema
-category: Domain Specific
-topic:	5.7 Grants, Loans & Subsidies
-subject: Payments Clearing and Settlement
-organisation: threesixtygiving
 name: 360Giving Schema
-status: review
-dateAdded: 2021-02-02
-dateUpdated: 2021-02-02
-classification: Domain Specific
+contraction: 360GS
+specification:
+  name: 360Giving
+  url: https://standard.threesixtygiving.org/en/latest/reference/#spreadsheet-format
+  published: "2018-06-01"
+license_id: CC-BY-4.0
+maintainer: threesixtygiving
+catalogue_status_events:
+- status: review
+  date: "2021-02-02"
+topics:
```

Comment:
could we make topics have a list of identifiers instead of free text? Also the numbers should go away. If it's better to address these changes in a separate PR it is fine but it needs cleaning

Reply:
I believe we said that we'd go for free-text, which could then be slugified for URL usage, as identifiers means we need to manage them (and increase contribution overhead)